content/ag-quiz.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Which Pa. attorney general candidate do you align with?"
+link-title = "AG Quiz"
+title-tag = "AG Quiz"
+description = "Pennsylvania’s April 23 primary election is fast approaching, and Democrats and Republicans will soon have to decide which attorney general candidate they want to send to the general election."
+image = "2024/03/01k0-8hfj-zvrj-6vd3.jpeg"
+image-description = "AG Quiz"
+image-credit = ""
+layout = "page-landing"
+signupHed = "Join our mailing list "
+signupDek = "Sign up to receive a weekly review of articles from Spotlight PA and occasional updates about our planning project in Berks County."
+url = "/elections-2024/candidate-quiz/"
+bg-img = true
+takeover = """
+{{<featured/main-takeover
+  title = "Which Pa. attorney general candidate do you align with?"
+  dek = ""
+  image = "2024/03/01k0-8hfj-zvrj-6vd3.jpeg"
+>}}
+
+{{</featured/main-takeover>}}
+"""
++++
+<div class="pt-9"></div>
+Pennsylvania’s April 23 primary election is fast approaching, and Democrats and Republicans will soon have to decide which attorney general candidate they want to send to the general election.
+
+The attorney general represents the interests of the state and the public, and defends Pennsylvania’s laws in court. The job has also served as a stepping stone for running for higher office in the commonwealth, including the governorship.
+
+This year, [five Democrats and two Republicans are running for the job](https://www.spotlightpa.org/news/2024/03/pennsylvania-election-2024-attorney-general-primary-candidates/). In many ways, the respective parties’ candidates are similar.
+
+All five Democratic candidates have vowed to defend abortion access and the state’s election results while promising to take a holistic approach to addressing violent crime. The two Republican candidates — both veterans who have been prosecutors — are running on tough-on-crime platforms and say the state constitution doesn’t guarantee a right to abortion.
+
+<div class="mt-9 md:-mx-20 shadow-md rounded-lg" data-tf-live="01HS8X1DTFCF3Q9KH0W69KG19W"></div><script src="//embed.typeform.com/next/embed.js?typeform-welcome=0"></script>
```

Comment:
I had to scroll to answer some of the questions and that confused the thingie and made it think I was skipping questions. Can we make the window taller?